vispy/app/backends/_ipynb_webgl.py

```diff
@@ -312,18 +312,30 @@ def _vispy_stop(self):
 
 
 # ---------------------------------------------------------- IPython Widget ---
+def _stop_timers(canvas):
+    """Stop all timers in a canvas."""
+    for attr in dir(canvas):
+        try:
+            attr_obj = getattr(canvas, attr)
+        except NotImplementedError:
+            attr_obj = None
+        if isinstance(attr_obj, Timer):
+            attr_obj.stop()
```

Comment:
Can you mention why you needed to do try/except, and put it in the comments? If I came to this code to modify it later I would be confused...